### PR TITLE
Ceph Mon Dump

### PR DESCRIPTION
**Pull request by shutstart**
This adds the endpoint to execute the `ceph-mon-dump` command to the ceph-api.

### CODE REVIEW

File: api/status.proto [hunk 1]
```diff
@@ -28,6 +30,52 @@ message GetCephStatusResponse {
   google.protobuf.Struct progress_events = 13; 
 }
 
+message CephMonDumpResponse {
+  int32 epoch = 1;
+  string fsid = 2;
+  string modified = 3;
+  string created = 4;
```
Comment (arttor) [inline api/status.proto:37]: could it be a [timestamp](https://protobuf.dev/reference/protobuf/google.protobuf/#timestamp) (same question for `modified`)?

**Review by arttor — CHANGES_REQUESTED** [review]

File: api/status.proto (hunk 1 above)
Comment (shutstart) [reply api/status.proto:37]: `{"created": "2024-10-07T14:56:11.421751+0200",
"modified": "2024-12-06T16:32:16.198494+0100",
"last_up_change": "2024-12-03T09:19:06.401376+0100",
"last_in_change": "2024-10-07T14:56:34.891449+0200",}`
All of them are in this format, not sure if this is how we can serialize in grpc

Comment (arttor) [reply api/status.proto:37]: i think that it is worth to parce string to Go [time.Time](https://pkg.go.dev/time#Parse) and then pass it to grpc timestamp. what do you think?

Comment (shutstart) [reply api/status.proto:37]: It's possible but would involve making a new struct and converting it to the grpc response seems worth it though.

**Review by arttor — APPROVED** [review]

**Pull request merged**